### PR TITLE
Added Emulating-Haiku-In-VMware-Workstation Article

### PR DESCRIPTION
**Pull request by vanishakesswani**

### CODE REVIEW

File: content/guides/virtualizing/Emulating-Haiku-In-VMware-Workstation.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+<div class="box-info">Running Haiku from a VM image is recommended to avoid any possible problems. Unless it does not work, or you would like to simulate an actual installation, do stick to this method.</div>
```
Comment (waddlesplash) [inline content/guides/virtualizing/Emulating-Haiku-In-VMware-Workstation.md]: This doesn't belong here; it belongs on the index page.

File: content/guides/virtualizing/Emulating-Haiku-In-VMware-Workstation.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+*   [Running Haiku from a VM image.](#part_vmimage)
+*   [Installing and running Haiku from an ISO image.](#part_iso)
+*   [Additional Steps.](#part_additional)
+*   [Troubleshooting.](#part_trouble)
+
+### Running Haiku from a VM image.
+
+##### Step 1. Downloading Haiku.
+
+Download the vmdk zip file from the Haiku website and unzip it. You should be able to see these following files. ![File system](/files/image1.jpeg)
```
Comment (waddlesplash) [inline content/guides/virtualizing/Emulating-Haiku-In-VMware-Workstation.md]: You'll need to add the images to the repository in the proper location and adjust all the paths appropriately.

File: content/guides/virtualizing/Emulating-Haiku-In-VMware-Workstation.md (hunk 1 above)
Comment (vanishakesswani) [inline content/guides/virtualizing/Emulating-Haiku-In-VMware-Workstation.md]: Removed .

File: content/guides/virtualizing/Emulating-Haiku-In-VMware-Workstation.md (hunk 2 above)
Comment (vanishakesswani) [inline content/guides/virtualizing/Emulating-Haiku-In-VMware-Workstation.md]: Done.

File: content/guides/virtualizing/vmware-workstation.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-<div class="box-info">Running Haiku from a VM image is recommended to avoid any possible problems. Unless it does not work, or you would like to simulate an actual installation, do stick to this method.</div>
++++ type = "article" title = "Virtualizing Haiku In VMware-Workstation" date = "2016-12-22T18:11:53.000Z" tags = [] +++
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-workstation.md]: These need to be split across multiple lines, as in other files.

File: content/guides/virtualizing/vmware-workstation.md [hunk 4]
```diff
@@ -13,11 +13,11 @@ The required ISO file can be found at on the "Get Haiku" page of this website wh
 
 ##### Step 1. Downloading Haiku.
 
-Download the vmdk zip file from the Haiku website and unzip it. You should be able to see these following files. ![File system](/files/image1.jpeg)
+Download the vmdk zip file from the Haiku website and unzip it. You should be able to see these following files. ![File system](https://www.haiku-os.org/files/image1.jpeg)
```
Comment (waddlesplash) [inline content/guides/virtualizing/vmware-workstation.md]: Uh, no, you need to add the files into the `static` directory in the proper subfolder, you can't just link to them on haiku-os.org.

**Comment by vanishakesswani** [comment]
@waddlesplash Made all required changes .

**Pull request merged**